crates/rpc/src/lib.rs

```diff
@@ -102,6 +110,10 @@ pub async fn handle_http_request(State(storage): State<Store>, body: String) ->
             let request = GetBlockByHashRequest::parse(&req.params).unwrap();
             block::get_block_by_hash(&request, storage)
         }
+        "eth_getBlockTransactionCountByNumber" => {
+            let request = GetBlockTransactionCountByNumberRequest::parse(&req.params).unwrap();
```

Comment:
remove unwrap in favor of `.ok_or(RpcErr::BadParams)?;`

Reply:
Not possible with the current api, should be fixed by #156